src/assets/index.js

```diff
@@ -1,31 +1,31 @@
-import parthmittal from "./parth.svg";
-import profilePic from "./profile-pic.jpg";
-import quotes from "./quotes.svg";
-import robot from "./robot.png";
-import send from "./Send.svg";
-import shield from "./Shield.svg";
-import star from "./Star.svg";
-import menu from "./menu.svg";
-import close from "./close.svg";
-import google from "./google.svg";
-import apple from "./apple.svg";
-import arrowUp from "./arrow-up.svg";
-import discount from "./Discount.svg";
-import facebook from "./facebook.svg";
-import instagram from "./instagram.svg";
-import linkedin from "./linkedin.svg";
-import twitter from "./twitter.svg";
+// import parthmittal from "./parth.svg";
+// import profilePic from "./profile-pic.jpg";
+// import quotes from "./quotes.svg";
+// import robot from "./robot.png";
+// import send from "./Send.svg";
+// import shield from "./Shield.svg";
+// import star from "./Star.svg";
+// import menu from "./menu.svg";
+// import close from "./close.svg";
+// import google from "./google.svg";
+// import apple from "./apple.svg";
+// import arrowUp from "./arrow-up.svg";
+// import discount from "./Discount.svg";
+// import facebook from "./facebook.svg";
+// import instagram from "./instagram.svg";
+// import linkedin from "./linkedin.svg";
+// import twitter from "./twitter.svg";
```

Comment:
Why comments? Thats why the build is failing

Applicable everywhere

Reply:
i thought rather to remove lines i should comment it out which can be used in future if required, so do i have to remove it like whole line?

Reply:
Remove the lines that are not required like binance, airbnb etc

But you have also commented out files that are required like insta, linkedIn etc